mage_ai/orchestration/db/models/oauth.py

```diff
@@ -192,7 +192,7 @@ def get_access(
         entity_id: Union[str, None] = None,
     ) -> int:
         permissions = []
-        if entity is None:
+        if entity == Permission.Entity.ANY:
```

Comment:
Will entity be None? What do we return for that case?

Reply:
It shouldn't be None, but I added a check for it just in case.